### DISCUSSION TITLE
Start in the Background

### DISCUSSION THREAD

**Discussion by lzsix** [Hofknecht/SystemTrayMenu#528] [Ideas]
![image](https://github.com/Hofknecht/SystemTrayMenu/assets/143028763/a4ecc119-92bb-43ed-97eb-a8961f21ddf3)

Implement a way to start the programm hidden/ in the background in order to not be visible in the taskbar all the time

**Comment by Miger88** [comment]
yes, this would be nice for me too.
And if visible, it woul be nice to change not only the Info icon in the Taskbar. Customizing the icon should change both and best would be only to have the second icon in my picture
![image](https://github.com/user-attachments/assets/ed1e48c8-4497-49c1-92d8-93286b8950e4)
